service/src/test/java/com/ge/predix/acs/zone/management/ZoneRepositoryTest.java

```diff
@@ -0,0 +1,124 @@
+package com.ge.predix.acs.zone.management;
+
+import java.util.Collections;
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTransactionalTestNGSpringContextTests;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import com.ge.predix.acs.config.InMemoryDataSourceConfig;
+import com.ge.predix.acs.testutils.TestActiveProfilesResolver;
+import com.ge.predix.acs.zone.management.dao.AttributeAdapterConnectionEntity;
+import com.ge.predix.acs.zone.management.dao.AttributeAdapterConnectionRepository;
+import com.ge.predix.acs.zone.management.dao.AttributeConnectorEntity;
+import com.ge.predix.acs.zone.management.dao.AttributeConnectorRepository;
+import com.ge.predix.acs.zone.management.dao.ZoneEntity;
+import com.ge.predix.acs.zone.management.dao.ZoneRepository;
+
+@EnableAutoConfiguration
+@ContextConfiguration(classes = { InMemoryDataSourceConfig.class })
+@ActiveProfiles(resolver = TestActiveProfilesResolver.class)
+public class ZoneRepositoryTest extends AbstractTransactionalTestNGSpringContextTests {
```

Comment:
Ok it's good to go. Integration tests are passing locally (Jenkins tests won't pass because of connections issues). Please merge so that we can continue with Rest API story.